src/components/EmailForm.tsx

```diff
@@ -31,21 +31,35 @@ export default function ({
           event.code === 'Enter' && emailIsValid ? onSubmit(email) : undefined
         }
       />
-      <GradientBorder>
+      {submitType === 'primary' ? (
```

Comment:
Hm, maybe it's better to leave as it was, we have too many codelines added only for `small` prop 🤔

Reply:
We have two different styled buttons, one for `/email`, and one for `/app`. If I leave it as it was when we would get a gradient border where it shouldn't show